File: atividade1.c
#include <stdio.h>

int main() {

  int a, b;

  printf("Digite o primeiro número:");
  scanf("%d", &a);
  printf("Digite o segundo número:");
  scanf("%d", &b);

  printf("Adição: %d + %d = %d\n", a, b, a + b);

  printf("Subtração: %d - %d = %d\n", a, b, a - b);

  printf("Multiplicação: %d * %d = %d\n", a, b, a * b);
 
  if (b != 0) {
    float divisao = (float)a / b;
    printf("Divisão: %d / %d = %.2f\n", a, b, divisao);
  } else {
    printf("Divisão por zero nao é permitida.\n");
  }
  
  return 0;
}

File: atividade1.java
import java.util.Scanner;

public class atividade1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite o primeiro número: ");
        double numero1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double numero2 = scanner.nextDouble();

        double soma = numero1 + numero2;
        double subtracao = numero1 - numero2;
        double multiplicacao = numero1 * numero2;
        double divisao = numero1 / numero2;

        System.out.println("A soma de " + numero1 + " + " + numero2 + " é igual a " + soma);
        System.out.println("A subtração de " + numero1 + " - " + numero2 + "é igual a" + subtracao);
        System.out.println("A multiplicação de " + numero1 + " * " + numero2 + "é igual a" + multiplicacao);
        System.out.println("A divisão de " + numero1 + " / " + numero2 + "é igual a" + divisao);

        scanner.close();
    }
}
